modules/rosa-sdpolicy-am-compute.adoc

```diff
@@ -14,7 +14,7 @@ All {product-title} clusters support a maximum of 180 worker nodes.
 
 [NOTE]
 ====
-The `Default` machine pool node type and size cannot be changed after the cluster is created.
+The `Default` machine pool node type or size cannot be changed after the cluster is installed.
```

Comment:
How about changing this passive sentence to an active sentence? Please skip this suggestion if the subject is varied?

> After the installation program installs the cluster, you cannot modify the `Defualt` machine pool node type or size. 

I do not see "machine pool node" in the OCP docs so maybe it should be "machine pool's node type or size"?

Reply:
Should the "delete a machine pool" statement be appended to this paragraph?

Reply:
"That" to "this" as "that" is a conjunction reserved for introducing restrictive clauses.

Reply:
Thank you for your suggestions. 
**For comment 1**:  
Your feedback is valid. However, ROSA docs have passive sentences at almost all the places. I believe, rather than changing one instance and to maintain consistency we can keep it in passive only or we can create a new Jira for the same, which can be a overhead task. What do you suggest?

**For comment 2 and 3:**:
I am not sure, how this comments are applicable for Line 17.  Are those intended for other lines? Could you please help clarify? Thank you!

Reply:
_machine pool node_ seems to be a ROSA term. I see 7 references in 7 files, all seem to be ROSA.

I don't mind the passive regarding the install, I think it is clear enough. 

> The machine config node type or size is configured upon installation. You cannot modify the default machine pool node type or size after the cluster is installed.

Reply:
@dfitzmau @mburke5678 Maybe:
After the cluster is installed, the `Default` machine pool cannot be deleted and its node type or size cannot be changed.